WordPress/Classes/Extensions/UIViewController+Notice.swift

```diff
@@ -32,4 +32,17 @@ extension UIViewController {
         }
         ActionDispatcher.dispatch(NoticeAction.post(notice))
     }
+    /// This method is will allow viewControllers to present an alert controller (action sheet style) that
```

Comment:
This doesn't feel like the right place for this as `UIViewController+Notice.swift` is really specific to making the `Notice` system work, not (yet) for making notices (or alerts) easier to display.

I'm not sure where a better place for this is. Maybe a new `UIAlert` extension, maybe on the tab bar (we use the tab bar for damn near everything).